libdirections-offboard/build.gradle

```diff
@@ -15,6 +15,7 @@ android {
 }
 
 dependencies {
+    implementation project(':libnavigation-util')
```

Comment:
Also unused at this point, right?

Reply:
no, here it'll be used. My next PR will change this module

Reply:
At this point unused though :upside_down_face: